### PR TITLE
Dockerfile Update: Add conditional db seeding to `init.dev.sh`

### LINKED ISSUE #180: DB not seeding on start
When starting the system via `./bin/start --pull` it seems the DB is not seeding properly, so I get an initial error:

Screenshot from 2020-10-08 16-27-30

When I exec into the website container and run:

`bundle exec rake db:reset db:migrate db:seed`

It goes through the motions, dropping, creating, seeding and then I am able to see the website:

Screenshot from 2020-10-08 16-29-27

(@iHiD can attest to this, he has waited for me several times while I have had to do this)

Is this seeding command being missed in the setup somewhere?

### PR DESCRIPTION
**Pull request by neenjaw**
Close #180

### CODE REVIEW

**Review by iHiD — APPROVED** [review]

**Comment by iHiD** [comment]
@ErikSchierboom One for you 🙂

File: docker/dev/init.dev.sh [hunk 1]
```diff
@@ -12,5 +12,11 @@ echo "Create and migrate DBs"
 mysql -u root -ppassword -h "$DB_HOST" < docker/dev/init.sql
 bundle exec bin/rails db:migrate
 
+num_users = $(rails r "p User.count")
```
Comment (iHiD) [inline docker/dev/init.dev.sh]:
```suggestion
num_users = $(bundle exec rails r "p User.count")
```

File: docker/dev/init.dev.sh [hunk 2]
```diff
@@ -12,5 +12,11 @@ echo "Create and migrate DBs"
 mysql -u root -ppassword -h "$DB_HOST" < docker/dev/init.sql
 bundle exec bin/rails db:migrate
 
+num_users = $(bundle exec rails r "p User.count")
```
Comment (neenjaw) [inline docker/dev/init.dev.sh]:
```suggestion
num_users=$(bundle exec rails r "p User.count")
```

**Comment by neenjaw** [comment]
Ok, this fix is good, but it is written for `bash`.  The docker `init.dev.sh` script is being run in the `sh` shell which doesn't support the same syntax.

**Comment by neenjaw** [comment]
It now seems to work as expected locally for me with this last script change.

**Review by iHiD — APPROVED** [review]

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
Thanks @neenjaw!

**Pull request merged**